tests/containers/docker_image.pm

```diff
@@ -23,32 +23,36 @@ use Mojo::Base qw(consoletest);
 use testapi;
 use utils;
 use containers::common;
+use containers::utils 'can_build_sle_base';
 use containers::container_images;
 use suse_container_urls 'get_suse_container_urls';
 use version_utils qw(get_os_release check_os_release);
 
 sub run {
     my ($image_names, $stable_names) = get_suse_container_urls();
     my ($running_version, $sp, $host_distri) = get_os_release;
-    my $runtime = "docker";
+    my $container_engine = "docker";
 
     install_docker_when_needed($host_distri);
-    allow_selected_insecure_registries(runtime => $runtime);
+    allow_selected_insecure_registries(runtime => $container_engine);
     scc_apply_docker_image_credentials() if (get_var('SCC_DOCKER_IMAGE'));
 
     for my $iname (@{$image_names}) {
-        test_container_image(image => $iname, runtime => $runtime);
-        build_container_image(image => $iname, runtime => $runtime);
+        record_info 'image name', '$iname';
+        test_container_image(image => $iname, runtime => $container_engine);
+        build_container_image(image => $iname, runtime => $container_engine) if can_build_sle_base;
```

Comment:
Container images can be build on non-SLE distribution too.

Reply:
yes but not sle base